### ISSUE TITLE
no longer works with new nrepl

### ISSUE THREAD

**Issue by sakue20** [bract/bract.dev#7]
```Caused by: java.io.FileNotFoundException: Could not locate clojure/tools/namespace/repl__init.class, clojure/tools/namespace/repl.clj or clojure/tools/namespace/repl.cljc on classpath.```

**Comment by kumarshantanu** [comment]
@sakue20 Which version of the new nREPL (and Clojure) are you using? Repro steps for this issue would be great to have.

**Comment by sakue20** [comment]
Using Clojure 1.10.0 and nREPL 0.6.0

`dev/user.clj`
```
(ns user
  (:require
    [clojure.repl     :refer :all]
    [bract.core.dev   :refer [start stop verbose config-files]]
    [bract.dev.reload :refer [go reinit reset restart]]))
```

`lein run` throws the error

**Comment by kumarshantanu** [comment]
I tried out the following steps but cannot reproduce the issue:

```
$ lein new gossamer foo
$ cd foo
$ # Edit dev/user.clj to add [clojure.repl     :refer :all]
$ lein do clean, with-profile uberjar run
```

Even adding the `:main` entry to root level in `project.clj` yielded the same result - no error.

Could you share your `project.clj` (dependency list) and the Leiningen version in use?

**Comment by kumarshantanu** [comment]
@sakue20 I've pushed `0.6.2-0.2.0-alpha1` to Clojars - in case the new version helps, let me know. /cc @vigneshsarma @nxvipin

**Comment by kumarshantanu** [comment]
Released version 0.6.2-0.2.0 GA (with Bract 0.6.2) - closing the issue, please feel free to reopen if it doesn't work.
